koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -472,6 +473,7 @@ def convert_csv_to_parquet(self, csv_filename):  # noqa: C901
         converters = self._get_column_converters()
         csv_path, csv_name = os.path.split(csv_filename)
         unique_keys = set()
+        aws_category_keys = set()
```

Comment:
I am already calling each subset call `category_keys`, so if I rename it it will muck with the logic a bit.
```
data_frame, data_frame_tag_keys, category_keys = data_frame
```
I am assuming you don't like the aws part from other comments, so let me update the vars to not much with the logic and remove the `aws`.